components/datepicker/src/auro-datepicker.js

```diff
@@ -1246,7 +1245,7 @@ export class AuroDatePicker extends AuroElement {
     const targetIsInput = initTarget.tagName === 'INPUT';
     const isFocusAlreadyOnInput = this.inputList.includes(this.shadowRoot.activeElement);
 
-    if (layoutRequiresHandling && !targetIsInput && !isFocusAlreadyOnInput) {
+    if (layoutRequiresHandling && !targetIsInput && !isFocusAlreadyOnInput && !event.composedPath().includes(this.dropdown.bibContent)) {
```

Comment:
I suspect event.composedPath doesn't work in NodeJS environment and will break the Borealis build. Probably need a try/catch on this.

Reply:
we talked about this on a call, determined it should be fine as is since it's 1. not executed in node, and 2. already being used by Borealis as of 1 month ago